test/PProf.jl

```diff
@@ -65,7 +65,7 @@ end
     end
     for i in 1:2
         if i == 1
-            data = Profile.fetch()
+            data = Profile.fetch(include_meta = false)
```

Comment:
tests are failing, and i'm guessing this is why. probably need the isdefined here too?

Reply:
ah haha you beat me to it. Perfect 💪